src/net.cpp

```diff
@@ -447,9 +447,8 @@ CNode* CConnman::ConnectNode(CAddress addrConnect, const char *pszDest, bool fCo
 
         // Look for an existing connection
         CNode* pnode = FindNode(static_cast<CService>(addrConnect));
-        if (pnode)
-        {
-            LogPrintf("Failed to open new connection, already connected\n");
+        if (pnode) {
+            LogPrintf("Failed to open new connection to %s, already connected (peer=%d)\n", addrConnect.ToString(), pnode->GetId());
```

Comment:
Maybe use `LogPrintfCategory` from the conflicting pull to avoid another breaking change in the log message later on?

Reply:
I'm happy to update the other pull (it's likely to see further updates) after this.

Reply:
Or can just close for now.

Reply:
No idea if that pull will be merged though, it has no Concept ACKs, will wait and see I guess.

Reply:
Yeah, it is pretty large. I think splitting up `LogPrintfCategory` on its own could make sense?

Reply:
It solves a few issues while at the same time reducing the amount of code, so I'm hopeful it will move forward, but yes, will probably begin splitting it up.

Reply:
> Maybe use `LogPrintfCategory`

Re-opening and will update.